src/Admin/AbstractAdmin.php

```diff
@@ -3531,6 +3515,38 @@ protected function configureDefaultSortValues(array &$sortValues)
     {
     }
 
+    /**
+     * Set the parent object, if any, to the provided object.
+     */
+    final protected function appendParentObject(object $object): void
```

Comment:
Do you need this method or can you make it private

Reply:
I don't need it. 
But since it's called in the `getNewInstance` and I know some users are overridden this methods (maybe without calling the parent method), I was thinking it could be useful to expose this methods.